src/JBCCIPSucker.sol

```diff
@@ -54,6 +57,36 @@ contract JBCCIPSucker is JBSucker, ModifiedReceiver {
         REMOTE_CHAIN_ID = IJBCCIPSuckerDeployer(msg.sender).REMOTE_CHAIN_ID();
         REMOTE_CHAIN_SELECTOR = IJBCCIPSuckerDeployer(msg.sender).REMOTE_CHAIN_SELECTOR();
         ROUTER = IRouterClient(i_ccipRouter);
+        // TODO: Remove this init and revert this back to 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2 for prod
+        WETH = CCIPHelper.wethOfChain(block.chainid);
+    }
+
+    /// @notice Map an ERC-20 token on the local chain to an ERC-20 token on the remote chain, allowing that token to be bridged.
+    /// @param map The local and remote terminal token addresses to map, and minimum amount/gas limits for bridging them.
+    function mapToken(JBTokenMapping calldata map) public override {
+        address token = map.localToken;
+        bool isNative = map.localToken == JBConstants.NATIVE_TOKEN;
+
+        /* // If the token being mapped is the native token, the `remoteToken` must also be the native token.
```

Comment:
is this to be deleted?

Reply:
yep. I'm making other changes to this function now that will prevent misconfigs.